spec/services/record_declaration_spec.rb

```diff
@@ -76,6 +76,40 @@
 
       it { is_expected.to be_valid }
     end
+
+    context "when `relevant_induction_record` is active" do
```

Comment:
I wonder if we can make the tests closer to what is actually happening, as in participant profile states without a cpd lead provider for active, but actual withdrawn states etc.?